rotkehlchen/chain/arbitrum_one/modules/kyber/constants.py

```diff
@@ -0,0 +1 @@
+CPT_KYBER = 'kyber legacy'
```

Comment:
in arbitrum you don't have the legacy. Make it final. And since it will be common from all the chains move it to constant in evm